Sources_v3/StreamChatUI/ChatChannel/MessageComposerSuggestionsViewController.swift

```diff
@@ -9,17 +9,18 @@ open class MessageComposerSuggestionsViewController<ExtraData: ExtraDataTypes>:
     UIConfigProvider,
     UICollectionViewDelegate,
     UICollectionViewDataSource {
-    var heightConstraint: NSLayoutConstraint?
-    
     // MARK: - Underlying types
-    
+
     public enum State {
         case commands([Command])
         case mentions([String])
     }
 
     // MARK: - Property
-    
+
+    private var frameObserver: NSKeyValueObservation?
+    public var bottomAnchorView: UIView?
```

Comment:
tip: You don't need to be afraid of putting comments to your code such that others can understand what you mean. What is `bottomAnchorView ` and why we need it?